src/hardware/memory.cpp

```diff
@@ -623,7 +632,7 @@ class MEMORY final : public Module_base {
 		// A20 Line - PS/2 system control port A
 		WriteHandler.Install(0x92, write_p92, io_width_t::byte);
 		ReadHandler.Install(0x92, read_p92, io_width_t::byte);
-		MEM_A20_Enable(false);
+		InitA20();
```

Comment:
Maybe I'm missing something, but is it possible to set `memory.a20.enabled = true;` before calling `MEM_A20_Enable(false);`?

Reply:
Our ideas crossed-paths in the ether! (see below)

Reply:
I was considering doing that in the init, but I felt dirty doing that - it seems like a "code WTF?".